coderd/userauth.go

```diff
@@ -46,6 +46,7 @@ const (
 	userAuthLoggerName      = "userauth"
 	OAuthConvertCookieValue = "coder_oauth_convert_jwt"
 	mergeStateStringPrefix  = "convert-"
+	oneTimePasscodeDuration = 20 * time.Minute
```

Comment:
Nit: it's not the duration, it's the validity period.

```suggestion
	oneTimePasscodeValidityPeriod = 20 * time.Minute
```

Reply:
or `oneTimePasscodeTTL` or `oneTimePasscodeExpiry` :)